src/Root.tsx

```diff
@@ -0,0 +1,25 @@
+import {
+  Navigate,
+  Route,
+  HashRouter as Router,
+  Routes,
+} from 'react-router-dom';
+import { App } from './App';
+import People from './pages/People';
+import HomePage from './pages/HomePage';
+
+export const Root = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
```

Comment:
You can add all paths in the enum and use them in all places in your application.